files/es/web/http/headers/www-authenticate/index.html

```diff
@@ -70,13 +70,11 @@ <h2 id="Specifications">Specifications</h2>
  </tbody>
 </table>
 
-<h2 id="Browser_compatibility">Browser compatibility</h2>
-
-
+<h2 id="Browser_compatibility">Compatibilidad con navegadores</h2>
 
 <p>{{Compat("http.headers.WWW-Authenticate")}}</p>
 
-<h2 id="See_also">See also</h2>
+<h2 id="See_also">Vea también</h2>
```

Comment:
```suggestion
<h2 id="See_also">Véase también</h2>
```
De acuerdo a nuestra guía https://github.com/mdn/translated-content/tree/main/docs/es

en las traducciones recomendadas ponemos así.